ONSdigital/rm-collection-exercise-service
# When creating collection exercise create default action plans and casetypedefaults if not exists

# Motivation and Context
When a collection exercise is created, if the default action plans and case type default do not exist for the given survey, then create them

# What has changed
Add sequence for casetypedefault table
Create new action plans and casetypedefaults if they do not exist for survey at collection exercise execution

# How to test?
Create the first collection exercise for a survey, check that the correct action plans and case type defaults are created
Acceptance tests/Integration test should pass

# Links
[Trello](https://trello.com/c/rnSOtuA1/66-create-case-type-default-in-collection-exercise-for-new-survey)
[rasrm-acceptance-tests PR](https://github.com/ONSdigital/rasrm-acceptance-tests/pull/140)
